src/entities/activity/ui/items/CheckboxItem/RegularGrid.tsx

```diff
@@ -0,0 +1,84 @@
+import { useMemo } from 'react';
+
+import { RegularCheckboxOption } from './RegularCheckboxOption';
+import { CheckboxItem } from '../../../lib/types/item';
+
+import { Box } from '~/shared/ui';
+import { splitList, useCustomMediaQuery } from '~/shared/utils';
+
+type Props = {
+  options: CheckboxItem['responseValues']['options'];
+  itemId: string;
+  values: string[];
+
+  onValueChange: (value: string, isNoneAbove: boolean) => void;
+  replaceText: (value: string) => string;
+  isDisabled: boolean;
+};
+
+export const RegularGrid = ({
+  values,
+  onValueChange,
+  isDisabled,
+  replaceText,
+  options,
+  itemId,
+}: Props) => {
+  const { lessThanSM } = useCustomMediaQuery();
+
+  const [evenColumn, oddColumn] = useMemo(() => {
+    return splitList(options);
+  }, [options]);
```

Comment:
Why would you need to split it up?

Reply:
I don`t remember to be honest. It was implemented a long time ago.